src/styling/_globals.scss

```diff
@@ -0,0 +1,51 @@
+$document-fontsize: 16px;
```

Comment:
Why an *underscore_ in the name? is it a convention?

Reply:
Yeah, if files have to be imported in other files, they need an underscore. Refer here - https://sass-lang.com/documentation/at-rules/import